spot-oa/ui/js/components/DendrogramMixin.react.js

```diff
@@ -64,12 +64,23 @@ var DendrogramMixin = {
                                         .style('fill', null);
                         });
 
-      nodeEl.append('text')
-                      .attr('dx', n => n.depth===0 ? -10 : 10)
-                      .attr('dy', 3)
-                      .style('text-anchor', n => n.depth===0 ? 'end' : 'start')
-                      .attr('fill', 'black')
-                      .text(n => n.name);
+      const y1 = nodes.find(n => n.depth === 1).y;
```

Comment:
Yeah but still, we are getting empty nodes and by getting "nodes" empty we are going to get an error on the javascript console. Shouldn't we be validating that even on dates with no information?